### PR TITLE
Presentation: fix testing package docs

### PR DESCRIPTION
**Pull request by saskliutas**
Restore `presentation-testing` package reference documentation generation which was removed for quite some time. Script was removed because it was failing for `presentation-testing` library. Failure was caused because `betools docs` command was passing `--exclude **/*test*/**/*` to typedocs and it was matching `presentation-testing` source folder and excluded all source files.

Changed `betools docs` command to have ability to override the default exclude pattern. This way correct test folder can be specified for `presentation-testing` package and docs generation does not fail.

### CODE REVIEW

**Review by grigasp — APPROVED** [review]

**Comment by saskliutas** [comment]
@wgoehrig @calebmshafer

File: tools/build/bin/betools.js [hunk 1]
```diff
@@ -40,6 +40,9 @@ yargs.strict(true)
         "excludeGlob": {
           describe: "Specify a directory, filename, or pattern to be excluded"
         },
+        "testExcludeGlob": {
```
Comment (calebmshafer) [inline tools/build/bin/betools.js:43]: Why do you need a new `testExcludeGlob` rather than using the existing `excludeGlob`?

Comment (saskliutas) [reply tools/build/bin/betools.js:43]: Exclude for `**/*test*/**/*` is hardcoded and `excludeGlob` just appends to it. So to resolve my issue `excludeGlob` should override the default exclude `**/node_modules/**/*,**/*test*/**/*`. Of course that would be breaking change if some package uses `excludeGlob` and relies on exclude for `*test*` and `node_modules` to be added be docs script.

So I was trying to add away to override that hardcoded glob for tests without introducing any required changes for other packages using `docs` script. If we can assume that this script is used only by packages in `itwinjs-core` repo (and `appui`) I guess I can remove hardcoded exclude for tests and update docs script in package.json files to pass `excludeGlob` for test files.

Comment (calebmshafer) [reply tools/build/bin/betools.js:43]: ah okay, missed that we default to excluding that test folder.

**Review by calebmshafer — APPROVED** [review]

**Pull request merged**